README.rst

```diff
@@ -401,6 +401,7 @@ To add autopep8 as a plugin, add this repo definition to your configuration:
 
 .. _`pre-commit`: https://pre-commit.com
 
+If you don't use a configuration file for autpep8, you'll want to include either ``args: [--in-place]`` or ``args: [--diff]``.
```

Comment:
I think it would be better to rephrase this since the use of `--in-place` or `--diff` options should be recommended for pre-commit usage regardless of whether a configuration file is used or not. Here's my suggested text:

```suggestion
When using autopep8 with pre-commit, it is recommended to include either ``args: [--in-place]`` or ``args: [--diff]``.
```